Makefile

```diff
@@ -28,18 +28,16 @@ dependency_tests:
 	python -c "import cdktf_cdktf_provider_random"
 	python -c "import cdktf_cdktf_provider_aws"
 
+_test: image_tests code_tests dependency_tests
```

Comment:
`_test` is confusing with `test`, should we name it like `in_container_test`?

Reply:
Yes, `in_container_test` is much better. Done in 2412554